tests/utils/test_disjoint_embeddings.py

```diff
@@ -314,6 +346,90 @@ def testRasterEmbeddingSearchBasic(self):
                 len(set(value_list)), len(value_list), "embeddings overlap"
             )
 
+    def test_find_sublattice_embeddings_tile(self):
+        # Check function responds correctly to tile specification
+        topology = "chimera"
+        min_graph_rows = 1
+        S = nx.from_edgelist({(i, i + 1) for i in range(5)})  # 6 nodes
+        T = dnx.chimera_graph(min_graph_rows + 1)
+        tile = dnx.chimera_graph(min_graph_rows, node_list=list(range(1, 8)))
+        embs = find_sublattice_embeddings(
+            S, T, graph_rows=min_graph_rows, max_num_emb=float("Inf"), tile=tile
+        )
+        self.assertEqual(len(embs), 4)
+        nodes_used = {v for emb in embs for v in emb.values()}
+        self.assertEqual(len(nodes_used), S.number_of_nodes() * len(embs))
+        self.assertTrue(
+            all(n % 8 > 0 for n in nodes_used), "Every 8th node excluded by tile"
+        )
+        tile5 = dnx.chimera_graph(min_graph_rows, node_list=list(range(3, 8)))
+        embs = find_sublattice_embeddings(
+            S, T, graph_rows=min_graph_rows, max_num_emb=float("Inf"), tile=tile5
+        )
+        self.assertEqual(len(embs), 0, "Tile is too small")
+
+        S = tile
+        embs = find_sublattice_embeddings(
+            S,
+            T,
+            graph_rows=min_graph_rows,
+            max_num_emb=float("Inf"),
+            tile=tile,
+            embedder=lambda x: "without S=tile trigger error",
+        )
+        self.assertEqual(len(embs), 4)
+        nodes_used = {v for emb in embs for v in emb.values()}
+        self.assertEqual(len(nodes_used), S.number_of_nodes() * len(embs))
+
+        invalid_T = nx.complete_graph(5)  # Complete graph is not a valid topology
+        with self.assertRaises(ValueError):
+            find_sublattice_embeddings(
+                S, invalid_T, graph_rows=min_graph_rows, tile=tile
+            )
+
+        # small_T = dnx.chimera_graph(2,2)
```

Comment:
This test is still not working. Goal: create a source graph that is small enough to fit into target graph but too large for tiles so itr triggers corresponding raises.